engine/crates/integration-tests/tests/federation/basic/headers.rs

```diff
@@ -4,160 +4,6 @@ use engine_v2::Engine;
 use graphql_mocks::EchoSchema;
 use integration_tests::{federation::EngineV2Ext, runtime};
 
-#[test]
-fn test_default_headers() {
```

Comment:
Are there toml equivalents for these? (I can't read everything super carefully so may have missed them)